Editor/Scripts/DCCIntegration/AutoServerCreation/AutoServerCreation.cs

```diff
@@ -0,0 +1,43 @@
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Unity.MeshSync.Editor {
+
+[InitializeOnLoad]
+public static class AutoServerCreation {
+    private const string IS_FIRST_SESSION_CALL = "MeshSync.AutoServerCreation.FirstSessionCall";
+    private const string OPT_OUT               = "MeshSync.AutoServerCreation.OptOut";
+    
+    static AutoServerCreation() {
+        
+        if(AutoServerCreationSettings.instance.HasPromptedUser)
```

Comment:
I think when users install MeshSync normally (from Package Manager), they may not necessarily want to have this prompt.

If this process is mostly intended for "Blender to Unity" installation workflow, I would say that we should probably do two things when clicking the button in Blender.

1. Update manifest.json (what we have discussed about)
2.  Write a configuration file (ScriptableSingleton below) that tells Unity what to do after installing MeshSync package. This way, we can even skip showing the prompt process.

Reply:
I agree, will implement it as you mentioned

Reply:
I implemented this, the default value is true so we assume we don't need to use the prompt, unless there is a settings file with that has the hasPromptedUser set to false. The blender script copies that setting file to the expected location.